compiler/dyno/lib/uast/chpl-syntax-printer.cpp

```diff
@@ -182,14 +182,16 @@ struct ChplSyntaxVisitor {
 
   /*
    * helper to check if the called expression is actually a
-   * management kind. Helps FnCalls not to print () in this case
+   * reserved word. Helps FnCalls not to print () in this case
   */
-  bool isCalleeManagementKind(const AstNode* callee) {
+  bool isCalleeReservedWord(const AstNode* callee) {
     if (callee->isIdentifier() &&
       (callee->toIdentifier()->name() == "borrowed"
```

Comment:
I think we should use `== USTR("borrowed")` etc here (along with updating compiler/dyno/include/chpl/queries/all-global-strings.h )